crates/optimism/payload/src/builder.rs

```diff
@@ -327,13 +329,26 @@ where
             }
         }
 
-        let withdrawals_root = ctx.commit_withdrawals(state)?;
+        debug_assert!(ctx.attributes().payload_attributes.withdrawals.is_empty());
 
         // merge all transitions into bundle state, this would apply the withdrawal balance changes
         // and 4788 contract call
         state.merge_transitions(BundleRetention::Reverts);
 
-        Ok(BuildOutcomeKind::Better { payload: ExecutedPayload { info, withdrawals_root } })
+        // withdrawals root field in block header is used for storage root of L2 predeploy
+        // `l2tol1-message-passer`
```

Comment:
```suggestion
        // `L2L1MessagePasser`
```

Reply:
rust convention is to not use camel case, but camel case would match 1-1 with specs, is that would you mean?